tests/testthat/tests-axis.R

```diff
@@ -83,6 +83,21 @@ test_that("axis tracks append FALSE by default",{
   # expect_equal(sum(names(gs$side.1) == 'axis'), 2)
 })
 
+context("axis style arguments handled appropriately")
+
+test_that("par args sent to axis() end up in axis args",{
+  gs <- points(gsplot(), 1, 0) %>% axis(side=1, tcl = -0.25)
+  expect_equal(gs$side.1$axis[["tcl"]], -0.25)
+})
+
+test_that("special args given to axis are retained", {
+  gs <- points(gsplot(), 1, 0) %>% axis(side=1, n.minor = 4)
+  expect_equal(gs$side.1$axis[["n.minor"]], 4)
+  
+  gs <- points(gsplot(), 1, 0) %>% axis(side=1, tcl.minor = -0.136)
+  expect_equal(gs$side.1$axis[["tcl.minor"]], -0.136)
+})
+
```

Comment:
should we add a test that does something with axis style args in other places? `gs <- points(gsplot(), 1, 0, tcl=-0.4) %>% axis(side=1, n.minor = 4)`